q2_feature_table/_heatmap/_visualizer.py

```diff
@@ -148,6 +147,10 @@ def heatmap(output_dir, table: pd.DataFrame,
     heatmap_plot.ax_col_dendrogram.set_position([col.x0, hm.y0 + height, width,
                                                  col.height])
 
+    # https://stackoverflow.com/a/34697479/3776794
+    plt.setp(heatmap_plot.ax_heatmap.xaxis.get_majorticklabels(), rotation=90)
+    plt.setp(heatmap_plot.ax_heatmap.yaxis.get_majorticklabels(), rotation=0)
```

Comment:
Fun! Looks like this is related to seaborn matplotlib magick --- if the labels are long enough they are rotated correctly on the y-axis:

<img width="386" alt="screen shot 2017-09-12 at 9 22 35 pm" src="https://user-images.githubusercontent.com/274668/30359634-fb5e7d60-9800-11e7-953a-90c226739f04.png">

👍 for forcing it to always be rotated

Reply:
Interesting! I also noticed there's different behavior across seaborn versions too.